src/hotspot/share/gc/g1/g1CollectionSet.cpp

```diff
@@ -300,8 +309,9 @@ double G1CollectionSet::finalize_young_part(double target_pause_time_ms, G1Survi
   double predicted_base_time_ms = _policy->predict_base_time_ms(pending_cards, _g1h->young_regions_cardset()->occupied());
   // Base time already includes the whole remembered set related time, so do not add that here
   // again.
+  size_t bytes_to_copy = 0;
   double predicted_eden_time = _policy->predict_young_region_other_time_ms(eden_region_length) +
-                               _policy->predict_eden_copy_time_ms(eden_region_length);
+                               _policy->predict_eden_copy_time_ms(eden_region_length, &bytes_to_copy);
```

Comment:
`bytes_to_copy` is never used afterwards, so it can be removed (and `predict...` does not need to get it passed.